spec/sql_footprint_spec.rb

```diff
@@ -54,21 +64,20 @@
     end
 
     it 'dedupes the same sql' do
-      expect do
-        Widget.create!
-        Widget.create!
-      end.to change { described_class.lines.length }.by(+1)
+      Widget.create!
+      Widget.create!
+      expect(sql_lines).to eq(sql_lines.uniq)
```

Comment:
Can we count the occurrences of `INSERT INTO widgets`? This matches the implementation a bit to much for my liking.